docs/subnets/create-first-subnet.md

```diff
@@ -0,0 +1,208 @@
+
+## Installation
+
+The fastest way to install the latest Avalanche-CLI binary is by running the install script:
+
+```bash
+curl -sSfL https://raw.githubusercontent.com/ava-labs/avalanche-cli/main/scripts/install.sh | sh -s
+```
+
+The binary will be installed inside the `./bin` directory (relative to where the install command was run).
+
+You can run all of the commands in this tutorial by calling `./bin/avalanche`.
+
+You can also add the command to your system path by running
+
+```
+cd bin
+export PATH=$PWD:$PATH
+```
+
+If you add it to your path, you should be able to call the program anywhere with just `avalanche`. To add
+it to your path permanently, add an export command to your shell initialization script (ex: .bashrc for
+Ubuntu or .zshrc for Mac).
+
+For more detailed installation instructions, see [Avalanche-CLI Installation](install-avalanche-cli)
+
+## Create Your Subnet Configuration
+
+Your first Subnet will be an Ethereum Virtual Machine (EVM) based Subnet. Avalanche's Subnet fork of the
+EVM is known as Subnet-EVM. It supports airdrops, custom fee tokens, configurable gas parameters, and
+multiple stateful precompiles. To learn more, check out [Subnet-EVM](https://github.com/ava-labs/subnet-evm).
+The goal of your first command will be to create a Subnet-EVM configuration.
+
+The Subnet command suite provides a collection of tools for developing and deploying Subnets.
+
+The Subnet Creation Wizard will walk you through the process of creating your subnet. To get started, first
+pick a name for your Subnet. This tutorial uses `mySubnet`, but feel free to substitute that with any name
+you like. Once you've picked your name, run
+
+`avalanche subnet create mySubnet`
+
+The following sections will walk through each question in the wizard.
+
+### Choose Your VM
+
+Select `SubnetEVM`.
+
+### Enter Your Subnet's ChainID
+
+Choose a positive integer for your EVM-style ChainID.
+
+In production environments, this ChainID will need to be unique and not shared with any other chain. You can visit [ChainList](https://chainlist.org/) to verify that your selection is indeed unique. Because this is a development subnet, feel free to pick any number. We advise staying away from well-known ChainIDs such as 1 (Ethereum) or 43114 (Avalanche C-Chain) as those may cause issues with other tools.
+
+### Token Symbol
+
+Enter a string to name your Subnet's native token. The token symbol does not necessarily need to be unique. Example token symbols are AVAX, JOE, and BTC.
+
+### Subnet-EVM Version
+
+Select `Use latest version`.
+
+### Gas Fee Configuration
+
+This question determines how gas fees are set on your submit.
+
+Select `Low disk use / Low Throughput 1.5 mil gas/s (C-Chain's setting)`.
+
+### Airdrop
+
+Select `Airdrop 1 million tokens to the default address (do not use in production)`.
+
+This address's private key is well-known, so do not send any production funds to it. They would likely be stolen instantly.
```

Comment:
Nit: I thought it's `address'`  (without the s)

Reply:
Ehhh I know the grammar rule you're talking about, but I think it's more common to write/say `address's`.